react-search-component/my-app/src/SearchResult.jsx

```diff
@@ -0,0 +1,30 @@
+export default function SearchResult({ items, keywords }) {
+  const defaultDisplay = items.map((item, index) => (
+    <li key={index}>{item}</li>
+  ));
+  const itemFiltered = items.filter((item) =>
+    item.toLowerCase().includes(keywords.toLowerCase())
+  );
+  const resultDisplay = itemFiltered.map((item, index) => (
+    <li key={index}>{item}</li>
+  ));
+  const noMatchDisplay = 'No items match the filter';
+
+  const listResult = returnList();
+
+  function returnList() {
+    let result = null;
+    if (keywords === '') {
+      result = <ul>{defaultDisplay}</ul>;
+    } else {
+      if (itemFiltered.length === 0) {
+        result = <p>{noMatchDisplay}</p>;
+      } else {
+        result = <ul>{resultDisplay}</ul>;
+      }
+    }
+    return result;
```

Comment:
The above nested if statements are a bit confusing! 

Here's one way you could simplify: 

`let result = <ul>{defaultDisplay}</ul>;`

`if (keywords !== '') { result = <ul>{resultDisplay}</ul>};`

`if (itemFiltered.length === 0) { result = <p>{noMatchDisplay}</p>};`